docs/hsi.xml

```diff
@@ -0,0 +1,875 @@
+<?xml version="1.0"?>
+<!DOCTYPE book PUBLIC "-//OASIS//DTD DocBook XML V4.3//EN"
+               "http://www.oasis-open.org/docbook/xml/4.3/docbookx.dtd">
+<reference id="hsi">
+  <title>Host Security ID Specification</title>
+  <warning>
+    <para>
+      This specification is still in active development: it is incomplete,
+      subject to change, and may have errors; use this at your own risk.
+      It is based on publicly available information.
+    </para>
+  </warning>
+
+<!--
+  <info>
+    <authorgroup>
+      <author>
+        <personname>
+          <firstname>Richard</firstname>
+          <surname>Hughes</surname>
+        </personname>
+        <affiliation>
+          <orgname>Red Hat</orgname>
+        </affiliation>
+      </author>
+      <author>
+        <personname>
+          <firstname>Mario</firstname>
+          <surname>Limonciello</surname>
+        </personname>
+        <affiliation>
+          <orgname>Dell</orgname>
+        </affiliation>
+      </author>
+      <author>
+        <personname>
+          <firstname>Alex</firstname>
+          <surname>Bazhaniuk</surname>
+        </personname>
+        <affiliation>
+          <orgname>Eclypsium</orgname>
+        </affiliation>
+      </author>
+      <author>
+        <personname>
+          <firstname>Alex</firstname>
+          <surname>Matrosov</surname>
+        </personname>
+        <affiliation>
+          <orgname>Nvidia</orgname>
+        </affiliation>
+      </author>
+    </authorgroup>
+  </info>
+-->
+
+  <partintro>
+
+    <refsect1 id="introduction">
+      <title>Introduction</title>
+      <para>
+        Not all system vendors prioritize building a secure platform.
+        The truth is that <emphasis role="strong">security costs money</emphasis>.
+        Vendors have to choose between saving a few cents on a bill-of-materials
+        by sharing a SPI chip, or correctly implementing BootGuard.
+        Discovering security vulnerabilities often takes an external researcher
+        filing a disclosure.
+        These disclosures are often technical in nature and difficult for an
+        average consumer to decipher.
+      </para>
+      <para>
+        The Linux Vendor Firmware Service (LVFS) could provide some
+        <emphasis role="strong">easy-to-understand</emphasis> information to
+        people buying hardware.
+        The service already knows a huge amount of information about machines
+        from signed reports uploaded to the LVFS and from analyzing firmware binaries.
+        However this information alone does not explain firmware security to the
+        user in a way they can actually interpret.
+      </para>
+    </refsect1>
+
+    <refsect2 id="existing-tooling">
+      <title>Existing Tooling</title>
```

Comment:
Right now existing means "pre-fwupd 1.5.x tooling".

Maybe this section needs to be renamed to "Other tools"?

Reply:
Agree, fixed.